shell/components/nav/__tests__/TopLevelMenu.test.ts

```diff
@@ -429,45 +428,6 @@ describe('topLevelMenu', () => {
     expect(description4.text()).toStrictEqual('some-description4');
   });
 
-  it('should not "crash" the component if the structure of banner settings is in an old format', async() => {
```

Comment:
We're no longer resizing the component based on settings so this test wasn't relevant any longer.